features/promotion/applying_promotion_rules/receiving_discount_based_on_nth_order_for_guest.feature

```diff
@@ -0,0 +1,36 @@
+@applying_promotion_rules
+Feature: Receiving discount based on nth order
+    In order to pay less while an placing order
+    As a Guest
```

Comment:
We agreed to write `As a Visitor` for this feature. Once we discuss all actors and its usages we will standardize all the features.

File should be renamed to "receiving_discount_on_first_order_for_guest**_customer**.feature"